### PR TITLE
Small update library.md - add an encoder to the KB initialisation as an example

### PR DESCRIPTION
**Pull request by coreation**
## Problem

Solves [this issue](https://github.com/pinecone-io/canopy/issues/186) 

## Solution

n/a

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Infrastructure change (CI configs, etc)
- [x] Non-code change (docs, etc)
- [ ] None of the above: (explain here)

## Test Plan

n/a

### CODE REVIEW

**Review by izellevy — CHANGES_REQUESTED** [review]

File: docs/library.md [hunk 1]
```diff
@@ -121,7 +121,10 @@ To create a knowledge base, you can use the following command:
 ```python
 from canopy.knowledge_base import KnowledgeBase
 
-kb = KnowledgeBase(index_name="my-index")
+# Make sure the KnowledgeBase encoder uses the same embedding model as the one that is used to embed your content, change the example value according to your specific configuration
+encoder = OpenAIRecordEncoder(model_name="text-embedding-3-small")
```
[the remark this answers is not in the document]
Comment (coreation) [reply docs/library.md:125]: hey @izellevy thanks for the comment, do I understand it correctly to 
* undo the changes on the library.md file
* add the text you wrote to the "actual documentation"...is that the README.md file?

Or do you want me to keep the added lines, but add the OpenAIRecordEncoder to the imports?

[the remark this answers is not in the document]
Comment (izellevy) [reply docs/library.md:125]: Hey @coreation sorry for the confusion. Inside the library.md file.

<OLD EXAMPLE> (init without the record encoder)

Alternatively, ....

<NEW EXAMPLE> (init with record encoder. Also we need to add the import OpenAIRecordEncoder)

[the remark this answers is not in the document]
Comment (coreation) [reply docs/library.md:125]: Got it, so just make a new example with the init containing an explicitly defined record encoder, below the existing example, right?

[the remark this answers is not in the document]
Comment (izellevy) [reply docs/library.md:125]: Exactly 👍

[the remark this answers is not in the document]
Comment (izellevy) [reply docs/library.md:125]: @coreation any updates on that?

[the remark this answers is not in the document]
Comment (coreation) [reply docs/library.md:132]: @izellevy done! Sorry for the delay for a small update, work/life is hectic at the moment :)

**Pull request merged**